float64 frequency
float64 leg_magnitude
float64 hip_magnitude
uint8   mode

